### DISCUSSION TITLE
Github COPILOT Bug

### DISCUSSION THREAD

**Discussion by menonkrishna57** [community/community#29698] [Copilot Conversations]
**Description**: COPILOT made a code mistake the part under with was by copilot and there is an error which is shown in terminal
**Steps to reproduce**:
**Expected behavior**:\n shouldnt have been there like that
![image](https://user-images.githubusercontent.com/66489697/184922909-85d75ff1-409b-4499-96ac-374628309635.png)

**Comment by mazkdevf** [comment]
![image](https://user-images.githubusercontent.com/79049205/184934718-118c7c07-c6a7-4569-895a-c0ff8e8ba051.png)

https://stackoverflow.com/questions/67680296/syntaxerror-f-string-expression-part-cannot-include-a-backslash

try something like on the stackoverflow page

**Reply by menonkrishna57** [reply]
Thanks but i know how to fix that. I meant to submit a bug report, the code that had a bug in it was written by Github COPILOT

**Reply by mazkdevf** [reply]
Oh okay, well it is made so it reads your code and then something then compares and etc.

But I don't really know, I don't work for GitHub.

But I will hope that you will have a good day!
